nilearn/regions/signal_extraction.py

```diff
@@ -27,67 +27,62 @@ def _check_labels(labels_img, target_shape, target_affine):
         regions definition as labels. Encodes the region labels of the signals.
 
     target_shape : numpy.ndarray
-        Desired shape of labels image and mask.
+        Desired shape of label images.
 
     target_affine : numpy.ndarray
-        Desired affine of labels image and mask.
-
-    Returns
-    -------
-    state : boolean
-        Corresponding state. Is only true for non-empty masks.
+        Desired affine of label images.
 
     """
     if labels_img.shape != target_shape:
-        raise ValueError("labels_img and imgs shapes must be identical.")
+        raise ValueError("label img and imgs shapes must be identical.")
```

Comment:
Sorry,I think it's better to relate to variable names:
```suggestion
        raise ValueError("label_imgs.shape and target_shape must be identical.")
```